packages/web-app-files/src/composables/keyboardActions/useKeyboardActionsSearchTable.ts

```diff
@@ -0,0 +1,162 @@
+import { computed, onBeforeUnmount, onMounted, ref, unref } from 'vue'
+import { useStore } from 'web-pkg/src/composables'
+import { useScrollTo } from 'web-app-files/src/composables/scrollTo'
+import { Key, ModifierKey } from 'web-pkg/src/composables/keyboardActions'
+import { eventBus } from 'web-pkg'
+import { useGettext } from 'vue3-gettext'
+
+export const useKeyboardActionsSearchTable = (keyActions, paginatedResources) => {
```

Comment:
Is this really SearchTable specific?

Reply:
So here's a thing. At this point, the goal is to provide a search table with actions in a new way.
Technically the search table is the same as a normal table, except you don't have defined space and have fewer actions you can perform. 

This means we can later get rid of `KeyboardAction.vue`, and split it into 2 composables or some abstraction that would combine `paginatedResources` action only and `space` related actions only. 
Then it all depends on personal preference if you want to create a composable that would define each table or you want to define different combinations in the setup of component

Reply:
> So here's a thing. At this point, the goal is to provide a search table with actions in a new way.

Yes 👍  The scope of this PR is fine as it is, I wouldn't change that. Let's get it merged first.

The next step after that IMO is to refactor `KeyboardAction.vue` into a separate composable like this one (it would be something like `useKeyboardActionsGenericSpaceTable` then) and use in in `GenericSpace.vue`.

After that we can determine the exact differences between both composables and decide if we want to try to combine those two, or if we need to abstract some more logic.